doc/classes/CubeMap.xml

```diff
@@ -1,8 +1,10 @@
 <?xml version="1.0" encoding="UTF-8" ?>
 <class name="CubeMap" inherits="Resource" category="Core" version="3.0.alpha.custom_build">
 	<brief_description>
+		A CubeMap is a 6 sided 3D texture.
 	</brief_description>
 	<description>
+		A CubeMap is a 6 sided 3D texture typically used for faking reflections. It can be used to make an object look as if it's reflecting it's surroundings. This usually delivers much better performace than other reflection methods.
```

Comment:
typos:
- performace -> performance
- reflecting it's surroundings -> reflecting its surroundings

Reply:
Thanks, both fixed.